src/main/frontend/handler/events.cljs

```diff
@@ -85,6 +97,20 @@
   (when-let [repo (get-local-repo)]
     (state/set-modal! (ask-permission repo))))
 
+
+
+(defmethod handle :after-db-restore [[_ repo]]
+  (println ":after-db-restore: " repo)
+  (let [conn* (conn/get-conn repo)
+        ast-version (:v (first (d/datoms conn* :aevt :ast/version)))]
+    (when (or (nil? ast-version)
+              (. semver lt ast-version db-schema/ast-version))
+      (notification/show!
+       [:p.content
+        (util/format "DB-schema updated, Please re-index repo [%s]" repo)]
+       :error
```

Comment:
We might need to display a `:warning`, otherwise, the user could be nervous. 😄

Reply:
updated!